src/state/storage.rs

```diff
@@ -0,0 +1,53 @@
+//! storage variables
+
+use std::collections::BTreeSet;
+
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Deque, Item, Map};
+
+use super::{
+    rate_limit::RateLimit,
+    rbac::{QueuedMessage, Roles},
+};
+
+/// Only this address can manage the contract. This will likely be the
+/// governance module, but could be set to something else if needed
+pub const GOVMODULE: Item<Addr> = Item::new("gov_module");
+/// Only this address can execute transfers. This will likely be the
+/// IBC transfer module, but could be set to something else if needed
+pub const IBCMODULE: Item<Addr> = Item::new("ibc_module");
```

Comment:
I don't see any usage of this storage item but in tests. seems unnecessary, or am I missing something?